modules/provision/container.go

```diff
@@ -93,10 +92,15 @@ func ContainerProvision(ctx context.Context, reservation Reservation) (interface
 		)
 	}
 
+	containerID, err := HexHash(reservation)
+	if err != nil {
+		return nil, err
+	}
+
 	id, err := containerClient.Run(
-		reservation.Tenant.String(),
+		containerID,
 		modules.Container{
-			Name:   uuid.New().String(),
+			Name:   reservation.User,
```

Comment:
This can cause conflict if the same user is hosting multiple containers on the same node. I suggest using the containerID.

Reply:
yep this is mistake, I inverted container tenant namespace and container name